resources/analysisTools/sophiaworkflow/safe_paste.py

```diff
@@ -69,6 +71,9 @@ def all_files_empty(all_data: list):
     if not all_files_empty(all_data):
```

Comment:
if you introduce the else block now, would it be reasonable to start the if statement without negation or do you prefer to still have the not-empty-block first?

Reply:
I can switch blocks. Some code style checkers also advise that. The only thing I do not like is `if ... return` blocks **without** `else` -- and then the rest of the method is `else`.

Reply:
but this would not be the case here, right?